File: 프로그래머스/0/181910. 문자열의 뒤의 n글자/문자열의 뒤의 n글자.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int n) {
    string answer = "";
    
    for (int idx = my_string.size() - n; idx < my_string.size(); idx++)
        answer.push_back(my_string[idx]);
    return answer;
}

File: 프로그래머스/0/181910. 문자열의 뒤의 n글자/문자열의 뒤의 n글자.py
def solution(my_string, n):
    answer = ''
    # for idx in range(len(my_string) - n, len(my_string)):
        # answer.append(my_string[idx])
        
    answer = my_string[len(my_string) - n : len(my_string)]
    return answer